packages/ui/src/FilesTable/index.tsx

```diff
@@ -136,6 +140,26 @@ export const FilesTable = ({
         return date.toLocaleDateString("en-GB");
       },
     },
+    {
+      field: "downloadCount",
+      header: translationMessage?.downloadCountColumnHeader || "Download count",
+      hidden: !visibleColumns.includes("downloadCount"),
+      body: (data) => {
+        return data.downloadCount;
+      },
+    },
+    {
+      field: "lastDownloadedAt",
+      header:
+        translationMessage?.lastDownloadedAtColumnHeader ||
+        "Last downloaded at",
+      hidden: !visibleColumns.includes("lastDownloadedAt"),
+      body: (data) => {
+        const date = new Date(data.lastDownloadedAt);
+
+        return date.toLocaleDateString("en-GB");
+      },
+    },
```

Comment:
lastDownloadedAt can be null.
```suggestion
      body: (data) => {
      if (!data.lastDownloadedAt) return "something";
      
        const date = new Date(data.lastDownloadedAt);

        return date.toLocaleDateString("en-GB");
      },
    },
```